src/components/Doc/Value.js

```diff
@@ -27,13 +27,13 @@ const Values = ({ name, values, defaultValue }) => {
             (defaultValue === false && valueContent === 'false');
           if (
             values.length === 1 &&
-            valueContent === 'string' &&
-            defaultValue
+            (valueContent === 'string' || valueContent === 'number') &&
+            defaultValue !== undefined
```

Comment:
In cases where defaultValue is zero, this condition will fail, hence it should be testing for undefined.